tasmota/tasmota_xdrv_driver/xdrv_50_filesystem.ino

```diff
@@ -925,7 +925,7 @@ uint8_t UfsDownloadFile(char *file) {
   UfsData.download_busy = true;
   char *path = (char*)malloc(128);
   strcpy(path,file);
-  BaseType_t ret = xTaskCreatePinnedToCore(donload_task, "DT", 6000, (void*)path, 3, nullptr, 1);
+  BaseType_t ret = xTaskCreatePinnedToCore(download_task, "DT", 6000, (void*)path, 3, nullptr, 1);
   if (ret != pdPASS)
```

Comment:
Why not free it here? Makes reading code a lot easier.

Reply:
Because it's a buffer that is passed from the main task to another
There is no way the main task knows when the other task is done
typical way of managing a buffer in a multi-task environment
I can add a comment to explain that if you want

Reply:
Nah. I get it now.